scrapers/PissPlay.yml

```diff
@@ -0,0 +1,32 @@
+name: PissPlay
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pissplay.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@id="video_title"]/text()
+      Date:
+        selector: //div[contains(@class, 'video_date')]
+        postProcess:
+          - parseDate: 2 Jan 2006
+      Performers:
+        Name:
+          fixed: Bruce and Morgan
+      Tags:
+        Name: //div[@id="video_tags"]/a
+      Details:
+        selector: //div[@id="video_description"]/text()
```

Comment:
are you sure about this?
most scenes i see (<https://pissplay.com/videos/myas-piss-bath>,<https://pissplay.com/videos/black-flag-review>,<https://pissplay.com/videos/playing-with-my-reflection>) have `<p>s` in the div
```yaml
        selector: //div[@id="video_description"]
```
or
```yaml
        selector: //div[@id="video_description"]/p
        concat: "\n\n"
```
seem to work

Reply:
God damit, that's my fault I did actually have the one I changed it to same as yours but committed from the wrong tab I had open which was an earlier work in progress. Farkin typical lol